docs/source/about/setup.rst

```diff
@@ -235,6 +235,12 @@ Install
    Sunshine needs access to `uinput` to create mouse and gamepad virtual devices and (optionally) to `uhid`
    in order to emulate a PS5 DualSense joypad with Gyro, Acceleration and Touchpad support.
 
+   #. Ensure that `uhid` is loaded at boottime.
```

Comment:
I took this section out of the new docs, as all of our provided packages were taking care of this.

Maybe we should re-add it, but to the "building" section?

Reply:
> Maybe we should re-add it, but to the "building" section?

Yes, I think that's a good idea. Also, in the new docs, there is still the setcap step left, which could also be moved to the building section as it's taken care of by the regular (i.e. not flatpak or appimage) packages.